Woof/WoofTests/ViewModel/LoginViewModelTests/LoginViewModelSetUserRoleTests.swift

```diff
@@ -0,0 +1,35 @@
+import XCTest
+
+final class LoginViewModelSetUserRoleTests: XCTestCase {
+    private var userPreferencesStorage = KeyValueStorage.testPreferencesStorage
+    private let testLoginViewModel = LoginViewModel()
+
+    override func setUp() {
+        super.setUp()
+        userPreferencesStorage.deleteData(for: KeyValueStorage.Key.userPreferences)
+    }
+
+    func testExistsInAPI() {
+        _ = testLoginViewModel.set(.sitter)
+    }
+
+    func testReturnsTrueWhenPassAllPossibleRoles() {
```

Comment:
The name of the test is not clear to me.
```suggestion

    func testReturnsTrueForEveryPossibleRoleWhenNewRoleWasSet() {
```
or 
```suggestion

    func testReturnsTrueWhenSetEveryPossibleRole() {
```

Reply:
fixed